pkgs/development/libraries/harfbuzz/default.nix

```diff
@@ -71,12 +79,9 @@ stdenv.mkDerivation {
 
   meta = with stdenv.lib; {
     description = "An OpenType text shaping engine";
-    homepage = "http://www.freedesktop.org/wiki/Software/HarfBuzz";
-    downloadPage = "https://www.freedesktop.org/software/harfbuzz/release/";
+    homepage = "https://github.com/harfbuzz/harfbuzz";
```

Comment:
The GH repo seems to point to https://harfbuzz.github.io/.

Reply:
Right.